test/core/security/evaluate_args_test.cc

```diff
@@ -32,9 +32,9 @@ class EvaluateArgsTest : public ::testing::Test {
 
 TEST_F(EvaluateArgsTest, EmptyMetadata) {
   EvaluateArgs args = util_.MakeEvaluateArgs();
-  EXPECT_EQ(args.GetPath(), nullptr);
-  EXPECT_EQ(args.GetMethod(), nullptr);
-  EXPECT_EQ(args.GetAuthority(), nullptr);
+  EXPECT_THAT(args.GetPath(), ::testing::IsEmpty());
```

Comment:
Please do not fully-qualify the namespace here, I don't believe it's necessary for compilation.

Reply:
I can make this change, but are you sure you want me to? This source file already consistently uses fully-qualified `::testing::` throughout, presumably to avoid ambiguity with `grpc::testing`.

Reply:
Ah, I didn't realize. That's fine, I can tackle it in a separate cleanup with the rest of the file.